maven-model-builder/src/site/apt/index.apt

```diff
@@ -183,13 +183,17 @@ Maven Model Builder
 | <<<project.baseUri>>>\
 <<<pom.baseUri>>> (<deprecated>) | the directory containing the <<<pom.xml>>> file as URI | <<<$\{project.baseUri\}>>> |
 *----+------+------+
+| <<<project.rootdir>>> | the project's root directory (containing a <<<.mvn>>> directory or with the <<<root="true">>> xml attribute | <<<$\{project.rootdir\}>>> |
+*----+------+------+
 | <<<build.timestamp>>>\
 <<<maven.build.timestamp>>> | the UTC timestamp of build start, in <<<yyyy-MM-dd'T'HH:mm:ss'Z'>>> default format, which can be overridden with <<<maven.build.timestamp.format>>> POM property | <<<$\{maven.build.timestamp\}>>> |
 *----+------+------+
 | <<<*>>> | user properties, set from CLI with <<<-Dproperty=value>>> | <<<$\{skipTests\}>>> |
 *----+------+------+
 | <<<*>>> | model properties, such as project properties set in the pom | <<<$\{any.key\}>>> |
 *----+------+------+
+| <<<session.rootdir>>> | the directory containing the root <<<pom.xml>>> file, identified by the existence of a <<<.mvn>>> child directory | <<<$\{session.rootdir\}>>> |
+*----+------+------+
```

Comment:
Where is the documentation for the top directory?

Reply:
`session.topdir` depends on the current directory so it should not be used during profile activation or model interpolation.  The `session.rootdir` property depends on `session.topdir` and thus should not be made available either.  Those two properties are available for profile injection, but this page is about the model building phase, which should be limited to `project.rootDirectory`.